tcms/testruns/templates/testruns/get.html

```diff
@@ -63,41 +63,41 @@ <h2 class="card-pf-title" style="text-align: left">
                     </h2>
                 {% endif %}
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-spinner"></span>{% trans 'Status' %}:
-                    <input class="bootstrap-switch" id="status_button" type="checkbox"
-                        {% if not object.stop_date %}checked{% endif %}
-                        {% if not perms.testruns.change_testrun %}disabled{% endif %}
-                        data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
-                </h2>
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
                     {{ object.planned_start|default:'-'}}
                 </h2>
 
+{#                Start date #}
                 <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
+                    {% if object.start_date == None %}
+                        <span class="fa fa-check"></span>{% trans 'Start date' %}:
```

Comment:
Fix this comment first - https://github.com/kiwitcms/Kiwi/pull/2328#discussion_r606700870; then 

- fa-calendar stays as an icon for the field itself
- fa-check becomes the actual text/icon for the button and "Start" becomes a hint which shows on mouse hover.